packages/liveblocks-react/src/__tests__/useThreads.test.tsx

```diff
@@ -1490,9 +1490,6 @@ describe("useThreads: error", () => {
 
     expect(result.current).toEqual({ isLoading: true });
 
-    // Unmount so polling doesn't interfere with the test
-    unmount();
```

Comment:
Can you explain why this was necessary?

Reply:
This unit test is checks if we retry with a backoff if there was an error while fetching threads. The fetcher looks like the following:

https://github.com/liveblocks/liveblocks/blob/734f5b6a8a40b2bbf2fc9b79699e3cc64e22c467/packages/liveblocks-react/src/umbrella-store.ts#L1336-L1377

If we unmount, the room is left and removed from the client in which case `client.getRoom(roomId)` fails and throws a `StopRetrying` error which is not retried unlike a fetching error, which interferes with the goal of the test.